src/components/MainParticipantInfo/MainParticipantInfo.tsx

```diff
@@ -108,6 +108,22 @@ const useStyles = makeStyles((theme: Theme) => ({
       background: '#A90000',
     },
   },
+  trackSwitchOffContainer: {
+    opacity: 0,
+    visibility: 'hidden',
+    transition: 'all 0.25s cubic-bezier(0.22, 0.61, 0.36, 1)',
+  },
+  isSwitchedOff: {
+    opacity: 1,
+    visibility: 'visible',
+    transition: 'all 0.5s linear 2s',
```

Comment:
Not sure about the 2s delay. This would display a frozen video for 2 seconds before it fades to black. Seems a little long. I'm not sure I like it because it means that the UI allows the user to be confused about the frozen video for 2 seconds before we reveal what is going on. I think we might just want to let the user know right away. A fade could still be fine though.